### PR TITLE
Remove RawData from exceptions.

### PR DESCRIPTION
**Pull request by brentschmaltz**
Add variables for nuspec
fix bug in building post in ProtocolMessage
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99%23discussion_r25113484%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99%23discussion_r25113956%22%2C%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99%23discussion_r25114134%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%20431dc72675ed193d9e306dddf984c2d5cd0019b2%20src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99%23discussion_r25113484%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Why%20are%20we%20removing%20this%20line%3F%20We%20validate%20this%20time%20stamp%20in%20K%20version.%22%2C%20%22created_at%22%3A%20%222015-02-20T23%3A49%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%2C%20%7B%22body%22%3A%20%22Scripts%20will%20add%20the%20this%20similar%20to%20%5Cr%5Cn%5C%22%5Bassembly%3A%20AssemblyInformationalVersionAttribute%28%5C%22%5C%22%24hash%5C%22%5C%22%29%5D%5C%22%22%2C%20%22created_at%22%3A%20%222015-02-21T00%3A00%3A04Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3172421%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/brentschmaltz%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222015-02-21T00%3A03%3A21Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/803796%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tushargupta51%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs%3AL34-38%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [x] <a href='#crh-comment-Pull 431dc72675ed193d9e306dddf984c2d5cd0019b2 src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs 4'></a> &nbsp;<b><a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99#discussion_r25113484'>File: src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs:L34-38</a></b>
- <a href='https://github.com/tushargupta51'></a> Why are we removing this line? We validate this time stamp in K version.
- <a href='https://github.com/brentschmaltz'></a> Scripts will add the this similar to
"[assembly: AssemblyInformationalVersionAttribute(""$hash"")]"

<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99?approve=1'></a>&nbsp;<a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by brandwe** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/99'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -34,4 +34,4 @@
 [assembly: NeutralResourcesLanguage("en-US")]
 [assembly: InternalsVisibleTo("Microsoft.IdentityModel.Protocol.Extensions.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100B5FC90E7027F67871E773A8FDE8938C81DD402BA65B9201D60593E96C492651E889CC13F1415EBB53FAC1131AE0BD333C5EE6021672D9718EA31A8AEBD0DA0072F25D87DBA6FC90FFD598ED4DA35E44C398C454307E8E33B8426143DAEC9F596836F97C8F74750E5975C64E2189F45DEF46B2A2B1247ADC3652BF5C308055DA9")]
 [assembly: AssemblyMetadata("Serviceable", "True")]
-[assembly: AssemblyFileVersion("1.0.REPLACE_FILE_VERSION_AT_BUILD_TIME")]
```
Comment (tushargupta51) [inline src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs]: Why are we removing this line? We validate this time stamp in K version.

Comment (brentschmaltz) [inline src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs]: Scripts will add the this similar to 
"[assembly: AssemblyInformationalVersionAttribute(""$hash"")]"

Comment (tushargupta51) [inline src/Microsoft.IdentityModel.Protocol.Extensions/Properties/AssemblyInfo.cs]: :+1:

**Pull request merged**